examples/todoapp/lib/base/models/response_models/deep_link_list_response_model.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2023, Prime Holding JSC
+// https://www.primeholding.com
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:json_annotation/json_annotation.dart';
+
+import '../deep_link_model.dart';
+
+part 'deep_link_list_response_model.g.dart';
+
+@JsonSerializable()
+class DeepLinkListResponseModel {
```

Comment:
As agreed all generated by the rx_bloc_cli files, which are not used need to be removed

Reply:
Done